File: cpp/bc2164.cpp
#include <iostream>
#include <cmath>
#include <iomanip>

using namespace std;

int main()
{
	int num;
	double fPa, sPa, ra = sqrt(5);
	
	cin >> num;

	fPa = pow((1 + ra) / 2, num);
	sPa = pow((1 - ra) / 2, num);
	
	cout << fixed << setprecision(1) << (fPa - sPa) / ra << endl;
}

File: python/bc2164.py
from math import pow, sqrt

num = int(input())

fPa = pow((1 + sqrt(5)) / 2, num)
sPa = pow((1 - sqrt(5)) / 2, num)
res = (fPa - sPa) / sqrt(5)

print(f"{res:.1f}")
